libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -2,59 +2,55 @@
 
 import android.animation.Animator;
 import android.animation.ObjectAnimator;
+import android.annotation.SuppressLint;
 import android.app.Dialog;
 import android.content.Context;
 import android.content.DialogInterface;
-import android.content.res.Configuration;
-import android.graphics.PorterDuff;
-import android.graphics.drawable.Drawable;
-import android.graphics.drawable.LayerDrawable;
 import android.os.Bundle;
 import android.os.CountDownTimer;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.view.animation.LinearInterpolator;
 import android.widget.FrameLayout;
+import android.widget.ImageButton;
 import android.widget.ImageView;
 import android.widget.ProgressBar;
-import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.core.content.ContextCompat;
-import androidx.core.graphics.drawable.DrawableCompat;
 import androidx.recyclerview.widget.GridLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.google.android.material.bottomsheet.BottomSheetBehavior;
 import com.google.android.material.bottomsheet.BottomSheetDialog;
 import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
 import com.mapbox.libnavigation.ui.R;
-import com.mapbox.navigation.ui.ThemeSwitcher;
+import com.mapbox.navigation.core.telemetry.events.FeedbackEvent;
 
-public class FeedbackBottomSheet extends BottomSheetDialogFragment implements FeedbackClickListener.ClickCallback,
-  Animator.AnimatorListener {
+import java.util.ArrayList;
+import java.util.List;
+
+public class FeedbackBottomSheet extends BottomSheetDialogFragment implements Animator.AnimatorListener {
 
   public static final String TAG = FeedbackBottomSheet.class.getSimpleName();
+  private static final String EMPTY_FEEDBACK_DESCRIPTION = "";
   private static final long CLOSE_BOTTOM_SHEET_AFTER = 150L;
   private static final long TIMER_INTERVAL = 1L;
-  private static final int LANDSCAPE_GRID_SPAN = 4;
-  private static final int PORTRAIT_GRID_SPAN = 2;
+  private static final int GRID_SPAN_GUIDANCE_LAYOUT = 2;
+  private static final int GRID_SPAN_NAVIGATION_LAYOUT = 3;
 
   private FeedbackBottomSheetListener feedbackBottomSheetListener;
-  private FeedbackAdapter feedbackAdapter;
-  private RecyclerView feedbackItems;
+  private RecyclerView guidanceIssueItems;
+  private FeedbackAdapter guidanceIssueAdapter;
+  private RecyclerView navigationIssueItems;
+  private FeedbackAdapter notificationIssueAdapter;
+  private ImageButton cancelBtn;
   private ProgressBar feedbackProgressBar;
```

Comment:
https://github.com/mapbox/navigation-sdks/issues/295#issuecomment-616760173
seems @asinghal22 is fine with keeping the auto dismiss functionality. will see how UX team replies to decide if we want to remove this code or not.